sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -212,7 +213,7 @@ def list_configuration_settings(
         """
 
     @distributed_trace
-    def list_configuration_settings(self, *args, **kwargs) -> ItemPaged[ConfigurationSetting]:
+    def list_configuration_settings(self, *args: Optional[str], **kwargs: Any) -> ItemPaged[ConfigurationSetting]:
```

Comment:
would a user actually pass None for args?

Reply:
That's allowed as the args are optional, but I don't think users will pass None for them.